client/src/controllers/DismissibleController.ts

```diff
@@ -0,0 +1,56 @@
+import { Controller } from '@hotwired/stimulus';
+import { WAGTAIL_CONFIG } from '../config/wagtailConfig';
+
+/**
+ * <section
+ *  aria-labelledby="whats-new-heading"
+ *  data-controller="w-dismissible"
+ *  data-w-dismissible-dismissed-class="w-dismissible--dismissed"
+ *  data-w-dismissible-id-value="{{ dismissible_id }}"
+ * >
+ * <div>
+ *  <button
+ *      type="button"
+ *      data-action="w-dismiss#dismissible"
+ *  >
+ *      <span class="w-sr-only">{% trans "Close" %}</span>
+ *  </button>
+ *  </div>
+ * </section>
+ */
+export class DismissibleController extends Controller<HTMLElement> {
+  static classes = ['dismissed'];
+
+  static values = {
+    id: { default: '', type: String },
+    dismissed: { default: false, type: Boolean },
+  };
+
+  declare dismissedClass: string;
```

Comment:
```suggestion
  declare readonly dismissedClass: string;
```

Minor, but more correct to say this is readonly.